packages/core/theme-module/theme/pages/MyAccount/OrderDetails.vue

```diff
@@ -0,0 +1,144 @@
+<template>
+  <section v-if="orderId" class="order-details">
+      <h2>Order #{{orderGetters.getNumber(order)}}</h2>
+      <p>Status: <strong>{{orderGetters.getStatus(order)}}</strong></p>
+      <p>Date: <strong>{{orderGetters.getDate(order)}}</strong></p>
+
+      <h3 class="title">Items</h3>
+      <SfCollectedProduct
+        v-for="product in orderGetters.getItems(order)"
+        :key="cartGetters.getItemSku(product)"
+        :image="cartGetters.getItemImage(product)"
+        :title="cartGetters.getItemName(product)"
+        :regular-price="'$' + cartGetters.getItemPrice(product).regular"
+        :qty="cartGetters.getItemQty(product)"
+        class="ordered-product"
+      />
+
+      <h3 class="title">Summary</h3>
+      <div class="order-details__summary">
+        <div class="order-details__summary__item">
+          <span class="property">Total price</span>
+          <span class="value">${{orderGetters.getPrice(order).regular}}</span>
+        </div>
+      </div>
+
+      <h3 class="title">Order addresses</h3>
+      <div class="order-details__addresses">
+        <div class="order-details__addresses__item">
+          <p class="order-details__addresses__item__title">Billing Address</p>
+          <div v-html="orderGetters.getBillingAddress(order)" />
+        </div>
+        <div class="order-details__addresses__item">
+          <p class="order-details__addresses__item__title">Shipping Address</p>
+          <div v-html="orderGetters.getShippingAddress(order)" />
+        </div>
+      </div>
+  </section>
+  <p v-else>Order not found</p>
+</template>
+
+<script>
+import { computed } from '@vue/composition-api';
+import { SfCollectedProduct } from '@storefront-ui/vue';
+import { onSSR } from '@vue-storefront/utils';
+import { useUserOrders, orderGetters, cartGetters } from '<%= options.composables %>';
+
+export default {
+  name: 'OrderDetails',
+  components: {
+    SfCollectedProduct
+  },
+  props: ['orderId'],
+  setup({ orderId }) {
+    const { searchOrders, orders } = useUserOrders();
+
+    onSSR(async () => {
+      await searchOrders({ id: orderId });
+    });
+
+    return {
+      cartGetters,
```

Comment:
I'm not entirely sure about that so its up for discussion but maybe we should just have item getters in `orderGetters` are we sure that in every platform item from order is the same one as cart item? Also from the DX point of view It doesn't seem intuitive, for every other comparable you have a single corresponding getter
@defudef @andrzejewsky whats your opinion

Reply:
I think we need in this case orderItemGetters with OrderItem component. Unfortunately, there is no component in SFUI for that case

Reply:
I'm not sure I get it, why you want to pair component and getters?

Reply:
The problem is with a component that renders bought items:
```
SfCollectedProduct
        v-for="product in orderGetters.getItems(order)"
        :key="cartGetters.getItemSku(product)"
        :image="cartGetters.getItemImage(product)"
        :title="cartGetters.getItemName(product)"
        :regular-price="'$' + cartGetters.getItemPrice(product).regular"
        :qty="cartGetters.getItemQty(product)"
        class="ordered-product"
      />
```
the orderGetters doesn't have methods that returns: SKU, image, or quantity

Reply:
RFC proposed here: https://github.com/DivanteLtd/next/issues/363